arclightning_backend/src/router.rs

```diff
@@ -28,12 +29,51 @@ fn list_files(path: PathBuf) -> Result<Vec<PathBuf>, io::Error> {
 #[derive(Debug, Clone)]
 pub struct Router {
     games_list: Arc<Mutex<HashMap<String, Game>>>,
+    static_dir: PathBuf,
+    access_key: Option<AccessKey>,
```

Comment:
this thing is gonna be accessed by requests so you probably want to wrap it with an arc mutex